Formula/k/kubectl-rook-ceph.rb

```diff
@@ -0,0 +1,30 @@
+class KubectlRookCeph < Formula
+  desc "Rook plugin for Ceph management"
+  homepage "https://rook.io/"
+  url "https://github.com/rook/kubectl-rook-ceph/archive/refs/tags/v0.8.0.tar.gz"
+  sha256 "b007bc9e971c253da53023f0448f82091d8dc94f8081789cf9b889e86d364a7a"
+  license "Apache-2.0"
+  head "https://github.com/rook/kubectl-rook-ceph.git", branch: "master"
+
+  livecheck do
+    url :stable
+    strategy :github_latest
+  end
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args(ldflags: "-s -w", output: bin/"kubectl-rook_ceph"), "cmd/main.go"
+  end
+
+  test do
+    output = shell_output("#{bin}/kubectl-rook_ceph --help")
+    assert_match("kubectl rook-ceph provides common management and troubleshooting tools for Ceph.", output)
+
+    output = shell_output("#{bin}/kubectl-rook_ceph health 2>&1", 1)
+    assert_match(
+      "Error: invalid configuration: no configuration has been provided, " \
+      "try setting KUBERNETES_MASTER environment variable\n", output
+    )
```

Comment:
this would be better syntax
```suggestion
    assert_match <<~EOS, shell_output("#{bin}/kubectl-rook_ceph health 2>&1", 1)
      Error: invalid configuration: no configuration has been provided
      try setting KUBERNETES_MASTER environment variable
    EOS
```

Reply:
> Minitest::Assertion: Expected /Error:\ invalid\ configuration:\ no\ configuration\ has\ been\ provided\ntry\ setting\ KUBERNETES_MASTER\ environment\ variable\n\n/ to match "Error: invalid configuration: no configuration has been provided, try setting KUBERNETES_MASTER environment variable\n".

This doesn't work and I'm not familiar with ruby so I just gonna revert it.